python/railgun.yml

```diff
@@ -0,0 +1,13 @@
+# This file is for Shopify employees development environment.
+# If you are an external contributor you don't have to bother with it.
+name: ci-queue
+
+vm:
+  image: /opt/dev/misc/railgun-images/default
+  ip_address: 192.168.64.245
+  memory: 1G
+  cores: 2
+volumes:
+  root: 512M
+services:
+   - redis
```

Comment:
why was this change needed?

Reply:
I was getting an error (`railgun.yml must exist in the same directory as dev.yml`) when running the tests. Based the change on this [commit](https://github.com/Shopify/ci-queue/commit/c998fc3a59fa2fd4909434b82837873e20e31645) that someone had started to also fix the python workflow issue. It's the same as the ruby [file](https://github.com/Shopify/ci-queue/blob/567f54f53fb1e4d65943b28f4742c2d566df4d66/ruby/railgun.yml) so hopefully these settings should be good.